packages/hawtio/src/plugins/camel/CamelContent.tsx

```diff
@@ -78,6 +79,15 @@ export const CamelContent: React.FunctionComponent = () => {
         !ccs.isEndpointsFolder(node) &&
         (ccs.isRouteNode(node) || ccs.isRoutesFolder(node)),
     },
+    {
+      id: 'routeDiagram',
+      title: 'Route Diagram',
+      component: RouteDiagram,
+      isApplicable: (node: MBeanNode) =>
+        !ccs.isEndpointNode(node) &&
```

Comment:
Checking it is not an Endpoint Node or Folder is probably now unnecessary given they are mutually exclusive to Route Node and Folder.

Reply:
I deleted that check.